Cartfile

```diff
@@ -1,4 +1,4 @@
-github "aws/aws-sdk-ios" ~> 2.14.0
+github "aws/aws-sdk-ios" ~> 2.16.0
```

Comment:
Eek, thanks for catching this.

We keep missing dependencies in the Cartfile, though. We need a mechanism to double-check that the Cartfile deps match those in the Podfile.

Reply:
we only ever bump this on minor version releases of the SDK, which is followed by an AppSync release so I got to this point from looking at the iOS release docs, and then over to the AppSync release docs. One place i can keep as a reminder is in the AppSync release docs, like "If this AppSync release is followed by the SDK minor release, do this" and have a reference to one of these PRs